test/unittests/eof_test.cpp

```diff
@@ -8,20 +8,20 @@
 
 using namespace evmone;
 
-TEST(eof, code_begin)
-{
-    EOF1Header header1{1, 0};
-    EXPECT_EQ(header1.code_begin(), 7);
-
-    EOF1Header header2{10, 0};
-    EXPECT_EQ(header2.code_begin(), 7);
-
-    EOF1Header header3{1, 1};
-    EXPECT_EQ(header3.code_begin(), 10);
-
-    EOF1Header header4{1, 10};
-    EXPECT_EQ(header4.code_begin(), 10);
-}
+// TEST(eof, code_begin)
```

Comment:
Why is this disabled?

Reply:
Now EOF1Header fields are filled by `read_valid_eof1_header`. We test this function in `TEST(eof, read_valid_eof1_header)` This commented test can be removed IMO.

Reply:
This was cheking `code_begin()` method, but now @chfast removed it, so the test can be removed, too.

Reply:
Removed.